index.js

```diff
@@ -1,18 +1,53 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+ const header = document.querySelector("#header")
 
+ console.log("Here's your header:", header)
+// console.log(`Here's your header:, ${header}`)
 
-/***** Deliverable 2 *****/
 
+/***** Deliverable 2 *****/
 
+header.style.color = "blue"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
 
 
-/***** Deliverable 4 *****/
+const travName = document.querySelector('h2')
+const travNickName = document.querySelector('em')
+const img = document.querySelector('img')
+
+console.log(traveler)
+travName.innerText = traveler.name
+travNickName.innerText = traveler.nickname
+img.src = 'pics/butterfly.jpg'
+img.alt = traveler.name
+console.log(img)
+
+console.log('This is what the traveler object looks like: ', traveler)
 
+// Using the traveler object, update the DOM to show the traveler's name
+
+
+
+/***** Deliverable 4 *****/
 
+animals = document.querySelector('#animals')
+
+traveler.animalSightings.forEach(function(animal){
+    let li = document.createElement("li")
+    li.dataset.id = animal.id
+    let p = document.createElement('p')
+    let img = document.createElement('img')
```

Comment:
you defined a global variable 'img' earlier in your code. try not to reuse variable names, especially global variables.